### PR TITLE
Add gallery of screens to the daily spending case study page

### PR DESCRIPTION
**Pull request by GBKS**
Adds an image gallery of the screens in the prototype so they can be more easily analyzed.

### CODE REVIEW

**Comment by pavlenex** [comment]
https://github.com/BitcoinDesign/Guide/pull/279/commits/8e89d04507304ed6773697974b16f011f8a7b53b
I like it. Really makes the page pop.

For the context, it's related to [this](https://deploy-preview-279--sad-borg-390916.netlify.app/guide/case-studies/cloud-backup/#design-details) particular part of the page, and here's how it [currently](https://bitcoin.design/guide/case-studies/cloud-backup/#cash-account--daily-spending) looks.

Screenshot 2021-05-04 at 18 31 42


@GBKS Thoughts on making these visuals consistent. I really like when it's[ slightly colored as in the onboarding page](https://github.com/BitcoinDesign/Guide/pull/225#issuecomment-822287210), or is the goal for this to be more of a wireframe/mock

**Comment by GBKS** [comment]
@pavlenex so @danielnordh created this prototype specifically in this wireframe style. Sometimes you don't want people to focus on design details, but on the overall flow, and this is a good method to get that effect. We could have a group discussion to see what the consensus is, but I think that's beyond this PR.

**Comment by danielnordh** [comment]
Agree, separate discussion. Let's not hold up this PR.
(I have started playing with using the UI Kit for these case studies, but that is on a different track)

**Comment by GBKS** [comment]
@danielnordh I copy/pasted the description text of each screen in that the image gallery. Some of them got a bit long. Would you want to revise the copy (and other copy one the page)? If so, this PR or a new one? What do you think of the header?

**Comment by pavlenex** [comment]
ping @danielnordh ^
@GBKS LGTM, ready for review?

**Comment by danielnordh** [comment]
The header ads some visual interest. It loaded slowly for me, not sure it's that heavy.
My only thought is that if we add a header on this page, the other case study pages feel quite different.
I would try to add them all in this PR, or make headers separate from the other changes.

**Comment by GBKS** [comment]
@danielnordh agreed. I'll remove the header from this one and split it out into a second PR that we can iterate on. Not convinced that the header I proposed here is that great.

**Comment by GBKS** [comment]
For a review, would be fantastic to get eyes on the captions of the newly added prototype screens. They are straight from Figma and I find them very long.

**Comment by GBKS** [comment]
Tightened up the caption copy. Good to go from my side. Text review would be great.

**Comment by GBKS** [comment]
@danielnordh good points. I don't like "Design details" either. Better if that carousel has a specific purpose that captures the core experience of the prototype. For example, we could call it "Onboarding sequence", only include those first screens, and add a very short summary in between the header and the carousel. The Figma link can then be the place to see all the screens in the prototype.

**Comment by GBKS** [comment]
@danielnordh I split up the carousel in two, focusing the first one on the onboarding experience via automatic cloud backup, and the second one on importing a wallet. They have more specific titles and descriptions. [Check it](https://deploy-preview-279--sad-borg-390916.netlify.app/guide/case-studies/cloud-backup/).

Looks like we're going to get a little bit of overlap between this page and the upcoming onboarding pages ([here](https://deploy-preview-302--sad-borg-390916.netlify.app/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/) and [here](https://deploy-preview-302--sad-borg-390916.netlify.app/guide/onboarding/backing-up-a-recovery-phrase/manual-backup/)).

File: guide/case-studies/cloud-backup.md [hunk 1]
```diff
@@ -42,4 +65,52 @@ A single-key scheme with [automatic cloud backup]({{ '/guide/private-key-managem
 - Additional user authentication to consider: biometrics, pin
 - Upgrade-path to other schemes if funds stored increase
 
+### Onboarding experience
+
+The wireframe screens below show the main onboarding sequence that guides users through the creation of multisig wallet.
```
Comment (danielnordh) [inline guide/case-studies/cloud-backup.md:70]:
```suggestion
The wireframe screens below show the main onboarding sequence that guides users through the creation of the wallet.
```

File: guide/case-studies/cloud-backup.md [hunk 2]
```diff
@@ -42,4 +65,52 @@ A single-key scheme with [automatic cloud backup]({{ '/guide/private-key-managem
...
+   height = 541
+   layout = "shadow"
+%}
+{% endfor %}
+
+</div>
+
+### Importing a wallet
+
+The import sequence guides the user through entering the recovery phrase, which is then also saved to cloud storage.
```
Comment (danielnordh) [inline guide/case-studies/cloud-backup.md:94]:
```suggestion
The import sequence guides the user through entering the recovery phrase, which is then also encrypted and saved to cloud storage.
```

**Review by danielnordh — APPROVED** [review]
I'm happy with this, makes the page more valuable without having to visit the Figma file.
Looks good to go.

**Pull request merged**